api/machine/machine.proto

```diff
@@ -747,8 +747,14 @@ message ControlPlaneConfig {
   string endpoint = 1;
 }
 
+message CNIConfig {
+  string name = 1;
+  repeated string urls = 2;
+}
```

Comment:
I wonder if one day we can generate protobufs from our go structs so that the two are always in sync.

Reply:
yeah, that can be nice... Do we have any existing means to generate protobufs from go structs?